src/github/pullRequestManager.ts

```diff
@@ -381,9 +381,9 @@ export class PullRequestManager {
 			});
 
 			const comments = data.repository.pullRequest.reviews.nodes
-				.map(node => node.comments.nodes.map(comment => this.addCommentPermissions(toComment(comment), remote)))
+				.map(node => node.comments.nodes.map(comment => this.addCommentPermissions(parseGraphQLComment(comment), remote)))

```

Comment:
can we consolidate the `parseGraphQLComment` and `toComment` methods into one?

Reply:
As in, I think we should completely replace `toComment` with `parseGraphQLComment`

Reply:
also since we get `viewerCanUpdate` and `viewerCanDelete`, we don't need to call `addCommentPermissions` here

Reply:
> can we consolidate the parseGraphQLComment and toComment methods into one?

yup and we should then always convert REST to GraphQL but not the other way around

> also since we get viewerCanUpdate and viewerCanDelete, we don't need to call addCommentPermissions here

good catch. BTW, do you think we should rename `canEdit` to `viewerCanUpdate`?

Reply:
`viewerCanUpdate` is a little bit clearer, but I think either are ok